src/platform/forms-system/src/js/fields/FileField.jsx

```diff
@@ -612,17 +607,15 @@ const FileField = props => {
             const fileNameId = `${idSchema.$id}_file_name_${index}`;
 
             if (hasVisibleError) {
-              setTimeout(() => {
-                scrollToFirstError();
+              setTimeout(async () => {
+                await scrollToFirstError();
                 if (enableShortWorkflow) {
                   const retryButton = $(`[name="retry_upload_${index}"]`);
                   if (retryButton) {
                     focusElement('button', {}, retryButton?.shadowRoot);
                   }
                 } else if (showPasswordInput) {
                   focusElement(`#${fileListId} .usa-input-error-message`);
-                } else {
-                  focusElement(ERROR_ELEMENTS.join(','));
```

Comment:
The `scrollToFirstError` applies focus to the error element, so the else case here was removed as redundant.